tests/src/system/basic/WskBasicNodeTests.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package system.basic
+
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+
+import common.JsHelpers
+import common.TestHelpers
+import common.TestUtils
+import common.Wsk
+import common.WskProps
+import common.WskTestHelpers
+import spray.json.DefaultJsonProtocol.StringJsonFormat
+import spray.json.pimpAny
+import spray.json.pimpString
+import common.TestUtils.RunResult
+import spray.json.JsObject
+
+@RunWith(classOf[JUnitRunner])
+class WskBasicNodeTests
+        extends TestHelpers
+        with WskTestHelpers
+        with JsHelpers {
+
+    implicit val wskprops = WskProps()
+    val wsk = new Wsk()
```

Comment:
Switches have been added to the test suites. You will have to do something like the below.

var usePythonCLI = true
val wsk = new Wsk(usePythonCLI)